kazoo/handlers/threading.py

```diff
@@ -17,16 +17,11 @@
 import threading
 import time
 
-import six
-
 import kazoo.python2atexit as python2atexit
 from kazoo.handlers import utils
 from kazoo.handlers.utils import selector_select
 
-try:
-    import Queue
-except ImportError:  # pragma: nocover
-    import queue as Queue
+import queue as Queue
```

Comment:
I would prefer you rename it `queue` instead of keeping the legacy capitalized name